content.md

```diff
@@ -26,17 +26,41 @@ A collection of tools and resources to enhance your gameplay!
 - [PP Rankings](https://pp.huismetbenen.nl/rankings/players/master) - Information and calculations for proposed and past pp reworks
 - [osu! World](https://osuworld.octo.moe/) - Provides provincial, state, and/or regional leaderboards
 ---
-### Beatmap Tools
+### Alternative Clients
+- [osu!lazer](https://github.com/ppy/osu) - Official rewrite of the game with a modern design, a bunch of new mods and features
```

Comment:
I wouldn't consider the effective replacement an alternative per se, seems a bit wrong to place it here.

Reply:
As I understand it, the plan is to eventually make lazer a proper update for stable, but at the moment  it's not feature complete, in constant development flux and not widely accepted by the community in it's current state. 

It will be a replacement eventually, but at the moment the stable client is still the default client for most people. That's why I added it as an "alternative". Once it becomes the main client, I'm sure we will remove it entirely.